### PR TITLE
we have a lunch menu!

### PR DESCRIPTION
**Pull request by jlkoran**

### CODE REVIEW

File: lunch.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+lunch:
```
Comment (Quetzalcohuatl) [inline lunch.txt]: capitalization??? please
LUNCH

File: lunch.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+1) pasta with tomato sauce and cheese
+2) b.l.t.
+3) ice cream
+4) chicken noodle soup
+5) salad with anchovies & croutons
+6) sliced anchovy ice cream with anchovies
```
Comment (atxu) [inline lunch.txt:8]: DELICIOUS!! where is 7) though

File: lunch.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+lunch:
+
+1) pasta with tomato sauce and cheese
+2) b.l.t.
+3) ice cream
+4) chicken noodle soup
+5) salad with anchovies & croutons
```
Comment (coxas) [inline lunch.txt:7]: nope @ anchovies

File: lunch.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+lunch:
+
+1) pasta with tomato sauce and cheese
+2) b.l.t.
+3) ice cream
```
Comment (ziqi-ma) [inline lunch.txt:5]: important

File: lunch.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+lunch:
+
+1) pasta with tomato sauce and cheese
+2) b.l.t.
```
Comment (oliviaweng) [inline lunch.txt:4]: Bacon lettuce and tuna

**Pull request merged**